samples/maps/demo/basic-map/demo.js

```diff
@@ -0,0 +1,73 @@
+const parseData = () => {
+    const lifeData = document.getElementById('csv').innerHTML;
+    const lines = lifeData.trim().split('\n');
+    lines.shift(); // Remove first line
+
+    const data = lines.map(line => {
+        const values = line.split(';');
+        if (values.length < 3 || isNaN(parseFloat(values[2]))) {
+            return null;
+        }
+        return {
+            code: values[1],
+            value: parseFloat(values[2])
+        };
+    }).filter(p => p); // Filter out null values
+    return data;
+};
```

Comment:
Since the purpose of this demo is to show how easy it is to use Highcharts Maps, I think we should remove this abstraction. Instead, just copy over the resulting data as JSON into the HTML file, and use that directly (through JSON.parse). Oh, and use `innerText` instead of `innerHTML`, as `innerHTML` poses a potential XSS threat.